cves/CVE-2017-9793.yml

```diff
@@ -139,19 +137,21 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
   answer:
-  date:
+    This vulnerability was reported by two employees from Huawei Technologies, Huijun Chen and Xiaolong Zhu,
+    who reported several different DoS vulnerabities found in multiple Apache Products.
+  date: 2017-09-05
   automated:
-  apache:
-  contest:
+  apache: false
+  contest: false
 
 subsystem:
   question: |
     What subsystems was the mistake in?
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: Dependencies XML
+  name: pom.xml
```

Comment:
I would look for a folder rather than an individual file for the subsystem.

Reply:
Fix should address that it is at the root.